transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__models.yml

```diff
@@ -28,67 +28,42 @@ models:
       Reconstructed `performance_events` table from schema `MM_TELEMETRY_PROD` using only columns needed by looker.
```

Comment:
I think we should also mention that the remaining columns have non-null values, as dictated by running the profiler at date X (e.g. March 21 2023) - at least until we have a regular profiler running job.

Reply:
The dashboard uses none of these fields, even if they were non-null in the future, it wouldn't apply to this use case.

Reply:
Does that mean that we remove columns that have non-null values as well? Or was the initial reason for removing them the fact that they were null? The title of the PR says that we remove the null columns, but the comment here say that we remove them because they are not referenced in Looker dashboards

Reply:
There's also some columns that have up to 8 non-null values (out of the 1+billion rows). These values are junk, making these columns practically always null.